### DISCUSSION TITLE
How to start wayfire automatically on boot

### DISCUSSION THREAD

**Discussion by Benik3** [WayfireWM/wayfire#2545] [Q&A] [answered]
Hi.
I have RPi running Home Assistant with DSI display.
I use headless RPi OS Lite and I installed wayfire on top of it to just show chromium browser in kiosk mode.
When I run `wayfire` from command line on the display, it correctly run and run also the browser.

But I'm not able to find a way, how to run the wayfire automatically, when the RPi boots.
I set that the tty1 on display even login automatically.

I tried to setup the wayfire by using service:
```
[Unit]
Description=Home assistant kiosk chromium
Requires=hassio-supervisor.service graphical.target
After=graphical.target hassio-supervisor.service

[Service]
Type=simple
#Restart=always
User=pi
Group=pi
PAMName=login
Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
Environment=XDG_RUNTIME_DIR=/run/user/1000
Environment=XDG_SESSION_TYPE=wayland

ExecStart=/usr/bin/dbus-run-session -- /usr/bin/wayfire -d -D

[Install]
WantedBy=graphical.target
```
What's weird, that in journalctl there is no error:
```
Dec 23 00:51:10 homeassistant systemd[1]: Started kiosk.service - Home assistant kiosk chromium.
Dec 23 00:51:10 homeassistant (-session)[16263]: pam_unix(login:session): session opened for user pi(uid=1000) by (uid=0)
Dec 23 00:51:20 homeassistant dbus-run-session[16298]: [32B blob data]
Dec 23 00:51:20 homeassistant dbus-run-session[16300]: [32B blob data]
Dec 23 00:51:21 homeassistant systemd[1]: kiosk.service: Main process exited, code=exited, status=255/EXCEPTION
Dec 23 00:51:21 homeassistant systemd[1]: kiosk.service: Failed with result 'exit-code'.
```
What is the correct way to run wayfire after boot?
I tried to google it, but without luck :(
Thanks!

**Comment by soreau** [comment, marked as answer]
There is a section on [the wiki](https://github.com/WayfireWM/wayfire/wiki/Tips-&-Tricks#system-unit) that might help.

**Reply by Benik3** [reply]
Thank you! I looked into wiki but I totally missed it (,,>﹏<,,)
I copied the example and it works fine.
Sorry for bothering.
